src/Algebra/Graph/Bipartite/AdjacencyMap.hs

```diff
@@ -709,6 +817,112 @@ biclique xs ys = let sxs = Set.fromList xs
                   in BAM (Map.fromSet (const sys) sxs)
                          (Map.fromSet (const sxs) sys)
 
+-- | The /star/ formed by a center vertex connected to a list of leaves.
+-- Complexity: /O(L log(L))/ time, where /L/ is the length of the given list.
```

Comment:
I think we can add the times sign to be coherent with the rest of the library (here and in others comments).

```suggestion
-- Complexity: /O(L * log(L))/ time, where /L/ is the length of the given list.
```

Reply:
Yes, we use `*` in the rest of the library.